src/overlays/gamestates/ovl_file_choose/z_file_choose_NES.c

```diff
@@ -4,96 +4,2362 @@
  * Description:
  */
 
-#include "z_file_choose.h"
+#include "file_select.h"
+#include "overlays/gamestates/ovl_opening/z_opening.h"
 #include "z64rumble.h"
+#include "z64save.h"
 #include "z64shrink_window.h"
 #include "z64view.h"
+#include "interface/parameter_static/parameter_static.h"
+#include "misc/title_static/title_static.h"
 
-extern UNK_TYPE D_01002800;
-extern UNK_TYPE D_01007980;
-extern UNK_TYPE D_0102A6B0;
-extern UNK_TYPE D_0102B170;
-extern UNK_TYPE D_010310F0;
-extern UNK_TYPE D_010311F0;
+s32 D_808144F10 = 100;
+f32 D_808144F14 = 8.0f;
+f32 D_808144F18 = 100.0f;
+s32 D_808144F1C = 0;
 
-// there are uses of D_0E000000.fillRect (appearing as D_0E0002E0) in this file
-extern GfxMasterList D_0E000000;
+static Gfx sScreenFillSetupDL[] = {
+    gsDPPipeSync(),
+    gsSPClearGeometryMode(G_ZBUFFER | G_SHADE | G_CULL_BOTH | G_FOG | G_LIGHTING | G_TEXTURE_GEN |
+                          G_TEXTURE_GEN_LINEAR | G_LOD | G_SHADING_SMOOTH),
+    gsDPSetOtherMode(G_AD_DISABLE | G_CD_MAGICSQ | G_CK_NONE | G_TC_FILT | G_TF_BILERP | G_TT_NONE | G_TL_TILE |
+                         G_TD_CLAMP | G_TP_NONE | G_CYC_1CYCLE | G_PM_1PRIMITIVE,
+                     G_AC_NONE | G_ZS_PIXEL | G_RM_CLD_SURF | G_RM_CLD_SURF2),
+    gsDPSetCombineMode(G_CC_PRIMITIVE, G_CC_PRIMITIVE),
+    gsSPEndDisplayList(),
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_file_choose/func_8080BC20.s")
+s16 sFileInfoBoxPartWidths[] = {
+    36, 36, 36, 36, 24, 28, 28,
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_file_choose/FileSelect_nop8080bc44.s")
+s16 sWindowContentColors[] = { 100, 150, 255 };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_file_choose/FileSelect_nop8080BC4C.s")
+s16 sFileSelectSkyboxRotation = 0;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_file_choose/func_8080BC58.s")
+s16 sWalletFirstDigit[] = {
+    1, // tens (Default Wallet)
+    0, // hundreds (Adult's Wallet)
+    0, // hundreds (Giant's Wallet)
+    0, // hundreds (Tycoon´s Wallet)
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_file_choose/func_8080BDAC.s")
+void FileSelect_IncrementConfigMode(FileSelectState* this) {
+    this->configMode++;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_file_choose/FileSelect_RenderView.s")
+void FileSelect_nop8080bc44(void) {
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_file_choose/func_8080BE60.s")
+void FileSelect_nop8080BC4C(FileSelectState* this) {
+}
```

Comment:
Err... in the event we leave the old VRAM in the function name, do we want capital hex or lower hex?
(personal preference upper hex)

Reply:
Yeah, upper hex makes sense. Although I just renamed this to Noop1 and Noop2, anyone can comment if they prefer the VRAM in there